models/cow_protocol/tx_hash_labels/offramp/ethereum/cow_protocol_tx_hash_labels_offramp_ethereum.sql

```diff
@@ -0,0 +1,38 @@
+{{
+    config(
+        alias='tx_hash_labels_offramp_ethereum',
+    )
+}}
+
+with
+ offramp_trades as (
+    select
+        tx_hash
+    from (
+        select tx_hash
+        from {{ ref('dex_aggregator_trades') }}
+        where blockchain = 'ethereum'
+        and token_bought_address in (select contract_address from {{ ref('tokens_ethereum_erc20_stablecoins') }})
+        and token_sold_address not in (select contract_address from {{ ref('tokens_ethereum_erc20_stablecoins') }})
+        UNION ALL
+        select tx_hash
+        from {{ ref('dex_trades') }}
```

Comment:
I see this union happening almost always... makes me wonder if this was the right way to go.

Reply:
> I see this union happening almost always... makes me wonder if this was the right way to go.

Definitely wasn't the optimal solution for someone like us who also compares themselves to other DEXs.